### ISSUE TITLE
Failed Loading React, PydioReactUI :  TypeError: Cannot read property 'nodeValue' of null

### ISSUE THREAD

**Issue by dansleboby** [pydio/pydio-core#1423]
Hello,
I get this error and I don't know why, if you can help me

Thanks
```
Failed Loading React, PydioReactUI :  TypeError: Cannot read property 'nodeValue' of null
    at Repository.loadFromXml (eval at <anonymous> ((index):1), <anonymous>:10:8541)
    at new Repository (eval at <anonymous> ((index):1), <anonymous>:10:5505)
    at User.loadFromXml (eval at <anonymous> ((index):1), <anonymous>:10:13856)
    at new User (eval at <anonymous> ((index):1), <anonymous>:10:10426)
    at Registry.parseUser (eval at <anonymous> ((index):1), <anonymous>:9:20654)
    at Function.eval (eval at <anonymous> ((index):1), <anonymous>:11:19781)
    at Pydio.Observable.notify (eval at <anonymous> ((index):1), <anonymous>:7:19562)
    at Pydio.fire (eval at <anonymous> ((index):1), <anonymous>:11:18973)
    at eval (eval at <anonymous> ((index):1), <anonymous>:11:20851)
    at eval (eval at <anonymous> ((index):1), <anonymous>:7:11338)
(anonymous) @ VM787:7
Promise.catch (async)
ResourcesManager.loadClassesAndApply @ VM787:7
(anonymous) @ VM787:11
Pydio.init @ VM787:11
(anonymous) @ pydio.boot.min.js?v=8.0.1:4
scriptLoaded @ pydio.boot.min.js?v=8.0.1:4
(anonymous) @ pydio.boot.min.js?v=8.0.1:4
Promise.then (async)
Connexion.loadLibrary @ pydio.boot.min.js?v=8.0.1:4
PydioBootstrap.refreshContextVariablesAndInit @ pydio.boot.min.js?v=8.0.1:4
PydioBootstrap.loadBootConfig @ pydio.boot.min.js?v=8.0.1:4
(anonymous) @ pydio.boot.min.js?v=8.0.1:4
trigger @ pydio.boot.min.js?v=8.0.1:3
onReady @ pydio.boot.min.js?v=8.0.1:3
```

**Comment by dansleboby** [comment]
When I go to page I see this:
![image](https://user-images.githubusercontent.com/4716382/40388506-be766ebe-5ddd-11e8-8c0d-f19728d9b3ea.png)

**Comment by cdujeu** [comment]
hi, please post in the forum, thanks! https://forum.pydio.com

**Comment by dansleboby** [comment]
I found the bug, it beacause 2 of my repo dosen't have a display name

**Comment by dansleboby** [comment]
I remove this and it fix the issue
![image](https://user-images.githubusercontent.com/4716382/40439172-2c1371e2-5e88-11e8-9aac-a773cfee2188.png)
But I don't know why this fix issue

**Comment by cdujeu** [comment]
hi, do you know how you ended up with those two un-labelled repositories? Did you create them via the admin panel?

**Comment by crotofinter** [comment]
Hi "cdujeu", same error that "dansleboby" user. It's a new installation with only three files, two users and one workspace. I had created the workspace via the admin panel. I could solve the error thanks to "dansleboby" user" with this SQL statement. It's Pydio 8, I have donwloaded the last version about 10 days ago.

![2018-11-05_16h44_26](https://user-images.githubusercontent.com/44779277/48008621-55d19d00-e11a-11e8-96f8-5b88974876f4.png)
